File: JAVA/hello.java
// javac: javac HelloWorld.java && java HelloWorld
// GraalVM Native: javac HelloWorld.java && native-image HelloWorld && ./helloworld
// GCJ (binary): gcj --main=HelloWorld HelloWorld.java -o hello && ./hello
// Maven: mvn compile exec:java (with pom.xml)
// Gradle: gradle run (with build.gradle)
// JAR: javac HelloWorld.java && jar cfe hello.jar HelloWorld HelloWorld.class && java -jar hello.jar
// jpackage: javac HelloWorld.java && jpackage --input . --main-jar hello.jar --main-class HelloWorld --name hello
// Excelsior JET: jet -main HelloWorld HelloWorld.java -o hello && ./hello

class HelloWorld
{
  public static void main(String[] args)
  {
    System.out.println("Hello World!!");
  }
}

File: src/SCALA/hello.scala
// Scala: scala hello.scala
// Scalac: scalac hello.scala && scala Hello
// SBT: sbt run
// SBT (JAR): sbt assembly && java -jar target/scala-*/hello-assembly-*.jar
// Mill: mill hello.run
// Scala Native: scala-native-cli package hello.scala && ./hello
// GraalVM Native: scalac hello.scala && native-image -cp . Hello && ./hello
// Scala.js: sbt fastOptJS && node target/scala-*/hello-fastopt/main.js


object HelloWorld extends App {
   println("Hello, World!")
 }
